tests/php/Dev/SapphireTestTest/DataProvider.php

```diff
@@ -42,34 +42,34 @@ public static function provideEqualListsWithEmptyList()
     public static function provideEqualLists()
     {
         return [
-            [
-                'oneParameterOneItem' => [
+            'oneParameterOneItem' => [
```

Comment:
Does this imply some potentially breaking changes to tests?

Reply:
In the end I've patched the underlying issue in sminnee/phpunit, but the issue is that if you do a call_user_func_array style construct, in PHP 8 string array keys are treated as named arguments.

Passing named arguments here was messy but benign in PHP7.

This was the fix in phpunit that avoids introducing named arguments so that PHP7 and PHP8 operation is the same.
https://github.com/sminnee/phpunit/commit/37220124361f4c1b8de6be7f7ca4d3ad3c479c7a